ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/host2.md

```diff
@@ -881,6 +881,7 @@ interface Dps1
 ```eos
 !
 interface Vxlan1
+   vxlan vlan 110 vni 10110
    vxlan vlan 111,113,115-118 vni 10111,10113,10115-10118
    vxlan vlan 111 flood vtep 10.1.1.10 10.1.1.11
    vxlan vlan 110 multicast group 239.9.1.4
```

Comment:
Please make sure this works on EOS - the output woudl be the combined range but just check that the input will be accepted.

Reply:
checked and ok